### PR TITLE
Multi entry model fix

### PR DESCRIPTION
**Pull request by BradleySappington**
Fix bug causing archive database update to save incorrectly and crash on failure.

### CODE REVIEW

**Comment by BradleySappington** [comment]
@mfixstsci @bhilbert4 Ready for review!

File: jwql/website/apps/jwql/archive_database_update.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def get_updates(inst):
                 latest_date = np.max(all_end_dates)
 
                 # Get the number of files in the observation
-                propobs = f'{proposal}{obsnum}'
+                propobs = f'jw{int(proposal):05}{obsnum}'
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/archive_database_update.py:122]: Ah. It was a missing leading 0 that was causing the trouble?

**Review by bhilbert4 — APPROVED** [review]
This looks good to me. Maybe in another PR we should create a tool that looks through the log files for this exception, so that we don't miss it if it occurs?

File: jwql/website/apps/jwql/archive_database_update.py (hunk 1 above)
Comment (BradleySappington) [reply jwql/website/apps/jwql/archive_database_update.py:122]: it checked for the string, but didn't verify the positioning,  so the string that it checked for existed in 2 files.  One where it was correctly the proposal and obs number, and another one where it was the end of the proposal, the obsnum, and the beginning of the visit number.
I'm now making sure we are checking from the very front of the name.

**Pull request merged**